blog/2024-01-09-introducing-sources/index.mdx

```diff
@@ -12,19 +13,19 @@ We’re excited to share a feature that unlocks a whole new level of power and f
+One of the most significant benefits of using our Stately editor is using your state machines as a source of truth. Using your state machines as live documentation relies upon bi-directionally syncing between visualizing your state machine and integrating the implementation code in your codebase. Previously, we had workarounds for bi-directional work; you could use the [XState VS Code extension](https://marketplace.visualstudio.com/items?itemName=statelyai.stately-vscode) or add your implementation code _after_ exporting your machine. You could even keep your machine in a separate file from your implementations to make it easier to replace your machine code with the exported code without overwriting your implementations. But you couldn’t provide implementation code in the editor, and importing your implementations from an existing machine would lose them. But that has all now changed with the introduction of sources!
 
-*Watch a demo of sources from our office hours at the end of December:*
+_Watch a demo of sources from our office hours at the end of December:_
 
-<YouTube id="x-F1TPjz_lI"/>
+<YouTube id="x-F1TPjz_lI" />
 
-*▶️ [Jump straight to the video chapter on sources](https://www.youtube.com/watch?v=x-F1TPjz_lI&t=318s).*
+_▶️ [Jump straight to the video chapter on sources](https://www.youtube.com/watch?v=x-F1TPjz_lI&t=318s)._
 
 ## What are sources?
 
 Using the <Code size={18} /> **Sources** panel, you can now provide source code for your actions, actors, and guards implementations from inside Stately’s editor.
 
-You can add custom source code from the **Sources** panel to suit your needs or choose from logic templates. Your source code will be included in your [live actors](/docs/stately-sky-getting-started), [synced projects](/docs/import-from-github), and [exported code](/docs/export-as-code), as found in the <Code size={18} /> **Code** panel. When an action, actor, or guard is selected, the **Sources** panel shows its source code. When no item is selected on the canvas, the  **Sources** panel shows all the sources for your current machine.
+You can add custom source code from the **Sources** panel to suit your needs or choose from logic templates. Your source code will be included in your [live actors](/docs/stately-sky-getting-started), [synced projects](/docs/import-from-github), and [exported code](/docs/export-as-code), as found in the <Code size={18} /> **Code** panel. When an action, actor, or guard is selected, the **Sources** panel shows its source code. When no item is selected on the canvas, the **Sources** panel shows all the sources for your current machine.
```

Comment:
Looks like Prettier took over and made many other changes. The extra spaces in the code I think is probably better. Are the other changes to markdown using the latest recommended syntax or should I revert these?